keps/sig-instrumentation/2305-metrics-cardinality-enforcement/kep.yaml

```diff
@@ -16,10 +16,12 @@ reviewers:
 approvers:
   - "@ehashman"
 creation-date: 2020-04-15
-last-updated: 2023-05-28
-stage: beta
+last-updated: 2024-01-20
+stage: stable
 status: implementable
-latest-milestone: "v1.29"
+latest-milestone: "v1.30"
 milestone:
-  beta: "v1.29"
+  alpha: "v1.21"
+  beta: "v1.28"
+  stable: "v1.30"
 disable-supported: true
```

Comment:
Do we still support disabling it?

Reply:
I don't think we have historically flipped that after reaching GA. For the next year at least, people will be using older versions so it's still relevant for those. Not very consistent though. FYI, looks like 94 of "stable" KEPs still say "true", 25 say "false" (not sure if they were *always* false though):

```console
[hi on] jbelamaric@jbelamaric:~/proj/gh/kubernetes/enhancements$ grep ^disable-supported $(grep -l '^stage: stable' $(find . -name kep.yaml)) | cut -d : -f 3 | sort | uniq -c
     25  false
     94  true
[hi on] jbelamaric@jbelamaric:~/proj/gh/kubernetes/enhancements$ 
```

Reply:
Generally available means you can't disable the feature gate, according to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages